docs/reference/docs/update-by-query.asciidoc

```diff
@@ -158,6 +158,8 @@ or omit to search all indices.
 include::{docdir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
 
 include::{docdir}/rest-api/common-parms.asciidoc[tag=analyzer]
++
+Defaults to `true`.
```

Comment:
I believe this is on the wrong parameter (analyzer).

However, when I tested `allow_no_indices` on the update by query API, it had no effect on whether an error was returned. Perhaps this should be removed?

Reply:
`localhost:9200/index-2*/_update_by_query?conflicts=proceed&allow_no_indices=false`:
```
{
    "error": {
        "root_cause": [
            {
                "type": "index_not_found_exception",
                "reason": "no such index [index-2*]",
                "resource.type": "index_or_alias",
                "resource.id": "index-2*",
                "index_uuid": "_na_",
                "index": "index-2*"
            }
        ],
        "type": "index_not_found_exception",
        "reason": "no such index [index-2*]",
        "resource.type": "index_or_alias",
        "resource.id": "index-2*",
        "index_uuid": "_na_",
        "index": "index-2*"
    },
    "status": 404
}
```

`localhost:9200/index-2*/_update_by_query?conflicts=proceed`:
```
{
    "took": 0,
    "timed_out": false,
    "total": 0,
    "updated": 0,
    "deleted": 0,
    "batches": 0,
    "version_conflicts": 0,
    "noops": 0,
    "retries": {
        "bulk": 0,
        "search": 0
    },
    "throttled_millis": 0,
    "requests_per_second": -1.0,
    "throttled_until_millis": 0,
    "failures": []
}
```

Reply:
Sorry about that. I made a mistake in my test setup.

Reply:
No worries at all, I didn't see your edit in time and tried it once more myself.

Reply:
However, we should still move the default value to the allow_no_indices parm.